apps/test/unit/code-studio/components/StageExtrasTest.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../util/reconfiguredChai';
+import StageExtras from '@cdo/apps/code-studio/components/stageExtras/StageExtras';
+import {bonusLevels} from './stageExtraTestHelpers';
+
+const DEFAULT_PROPS = {
+  stageNumber: 1,
+  nextStageNumber: 2,
+  nextLevelPath: '',
+  showProjectWidget: true,
+  projectTypes: [],
+  bonusLevels: bonusLevels,
+  sectionId: 3,
+  userId: 5,
+  showStageExtrasWarning: true
+};
+
+describe('StageExtras', () => {
+  it('does not show stage extras warning if showStageExtrasWarning is false ', () => {
```

Comment:
nit: remove trailing spaces from test case names, here and on the other `it(...)` lines below?